### PR TITLE
Homework12

### PR DESCRIPTION
**Pull request by YurySubbota**

### CODE REVIEW

File: homework12/homework_12.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+class Moving:
+    def __init__(self, name):
```
Comment (nicilytop-tms) [inline homework12/homework_12.py:2]: hmm.. is it name for ducks and tigers? 🙃

File: homework12/homework_12.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+class Transport(Moving):
+    def launch(self):
+        raise NotImplementedError('I do not understand this')
+
+
+class Duck(Animal):
+    def voice(self):
+        print('cria')
```
Comment (nicilytop-tms) [inline homework12/homework_12.py:21]: Is this Russian duck? 🤔🤣

**Review by nicilytop-tms — APPROVED** [review]
Nice job man 😎

File: homework12/homework_12.py (hunk 2 above)
Comment (YurySubbota) [reply homework12/homework_12.py:21]: Tak tochno major razvedki Marina Nikolaevna Utkina. pozivnoy Duck

**Pull request merged**